doc/manual/oqum/hazard/02c_output_event_based.tex

```diff
@@ -32,20 +32,9 @@
 rupture ID is an integer that represents each rupture uniquely: it should
 not be confused with the event ID.
 
-The outputs from the \glspl{acr:gmf} can be exported either in the xml or csv
-formats. If the \glspl{acr:gmf} are exported in the xml format, the
-\gls{acr:oqe} will produce an xml file for each realisation with the
-corresponding ground motion fields. Listing~\ref{lst:output_gmf_xml} is an
-example of a gmf collection NRML file containing one ground motion field:
-
-\begin{listing}[htbp]
-  \inputminted[firstline=1,firstnumber=1,fontsize=\footnotesize,frame=single,linenos,bgcolor=lightgray]{xml}{oqum/hazard/verbatim/output_gmf.xml}
-  \caption{Example ground motion field collection output file comprising a single GMF}
-  \label{lst:output_gmf_xml}
-\end{listing}
-
-Exporting the outputs from the \glspl{acr:gmf} in the csv format results in
-two csv files illustrated in the example files in
+The outputs from the \glspl{acr:gmf} can be exported in the csv
+format. Exporting the outputs from the \glspl{acr:gmf} in the 
+csv format results in two csv files illustrated in the example files in
```

Comment:
There is actually a third file sigma_epsilon.csv that was requested to use by Swiss RE.